build.sh

```diff
@@ -236,27 +239,60 @@ function build_qemu {
 }
 
 function build_odroid_h4 {
-  DEFCONFIG="configs/config.hardkernel_odroid_h4"
+  DEFCONFIG=$1
   FW_VERSION=$(cat ${DEFCONFIG} | grep CONFIG_LOCALVERSION | cut -d '=' -f 2 | tr -d '"')
 
   # checkout several submodules needed by these boards (some others are checked
   # out by coreboot's Makefile)
   git submodule update --init --force --checkout \
       3rdparty/dasharo-blobs
 
-  docker run --rm -t -u $UID -v $PWD:/home/coreboot/coreboot \
-    -v $HOME/.ssh:/home/coreboot/.ssh \
-    -w /home/coreboot/coreboot coreboot/coreboot-sdk:$SDKVER \
-    /bin/bash -c "make distclean"
+  if [ "${AIRGAP}" -eq 1 ]; then
```

Comment:
Should be applied to all boards, not just selected ones. Mayeb it would also be worth to separate it into a function to reduce the LOC and overall boilerplate.

Reply:
9260ef38e56c5d426927b5aa6a9bfaac5a1c83b4